woocommerce/wc-admin
# Check if tax was successfully added before notice

Fixes #6031 

Check if tax was successfully added before notice.


### Screenshots
Invalid row
<img width="149" alt="invalid" src="https://user-images.githubusercontent.com/56378160/106218958-762c1d00-61a6-11eb-83f3-7afb9e0e85c3.png">
Doesn't render notice after click save changes
<img width="1440" alt="no notice" src="https://user-images.githubusercontent.com/56378160/106218999-88a65680-61a6-11eb-8d3c-39380062b9ec.png">
First valid row
<img width="137" alt="螢幕快照 2021-01-28 下午8 18 06" src="https://user-images.githubusercontent.com/56378160/106219071-ad023300-61a6-11eb-8a79-a10b8fbc9d6b.png">
Render notice
<img width="487" alt="螢幕快照 2021-01-28 下午8 19 19" src="https://user-images.githubusercontent.com/56378160/106219126-c1463000-61a6-11eb-9a0b-51cb8943ab72.png">


### Detailed test instructions:
- Set `add_onboarding_tax_notice_admin_script` condition to true.
- Go to WooCommerce settings -> Tax -> Standard rates.
- Insert row with invalid country code and save changes.
- Check that the "You've added your first tax rate!” message does not pop up.

-----------
- Insert row with valid country code and save changes.
- Check that the "You've added your first tax rate!” message pops up.

-----------

- Insert the second row with a valid country code and save changes.
- Check that the "You've added your first tax rate!” message does not pop up.
